src/config/index.ts

```diff
@@ -216,6 +218,11 @@ export const OVERSEAS_ENTITY_PAYMENT_WITH_TRANSACTION_URL = UPDATE_AN_OVERSEAS_E
 export const RESUME = "resume";
 export const RESUME_SUBMISSION_URL = REGISTER_AN_OVERSEAS_ENTITY_URL + TRANSACTION_PATH + OVERSEAS_ENTITY_PATH + RESUME;
 
+// Update configs
+export const UPDATE_REVIEW_BENEFICIAL_OWNER_INDIVIDUAL_URL_WITH_PARAM_URL = UPDATE_REVIEW_BENEFICIAL_OWNER_INDIVIDUAL_URL + "?index=" + ID + "&review=true";
+export const REVIEW_BENEFICIAL_OWNER_INDEX_PARAM = "?index=";
+export const REVIEW_BENEFICIAL_OWNER_REVIEW_PARAM = "&review=true";
```

Comment:
really strange to have the true in the constant. Also use consts in line above and 206?